spec/es_spec_helper.rb

```diff
@@ -10,6 +10,19 @@ def get_host_port
   def get_client
     Elasticsearch::Client.new(:hosts => [get_host_port])
   end
+
+  def self.es_version_satisfies?(requirement)
+    es_version = ENV['ES_VERSION']
+    result = true
+    if es_version == "master"
```

Comment:
My worry with this strategy is that if there's a bug in this code we might accidentally not run tests that should run and get green builds. 

I'm not sure what the solution here is however.  We should make absolutely sure that the tests we think are running actually are running before we merge.

Once this PR is out of WIP and all tests are passing we should both check the CI logs and manually confirm that.